vertx/echo/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Although I created this pom, to be consistent with the original vert.x example, there should be no copyright.